### PR TITLE
docs: update install guide

### PR DESCRIPTION
**Pull request by davemooreuws**

### CODE REVIEW

File: src/pages/guides/getting-started/installation.mdx [hunk 1]
```diff
@@ -15,34 +15,58 @@ Please follow these links to the official installation steps for each.
...
 scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
 scoop install nitric
 ```
 
-### Linux
+```bash {{ title: 'Linux', label: 'nitric can be downloaded as a scripted install via curl.' }}
+curl https://nitric.io/install | bash
+```
+
+```bash {{ title: 'Brew', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}
```
Comment (HomelessDinosaur) [inline src/pages/guides/getting-started/installation.mdx:33]: Does brew need its own tab given it's provided under MacOS?

File: src/pages/guides/getting-started/installation.mdx [hunk 2]
```diff
@@ -15,34 +15,58 @@ Please follow these links to the official installation steps for each.
...
+brew install nitrictech/tap/nitric
+```
+
+</CodeGroup>
+
+### Manual Install
 
-`nitric` can be downloaded as a scripted install via curl.
+You can download pre-compiled binaries from the [releases](https://github.com/nitrictech/cli/releases) page and copy them into your desired location
```
Comment (HomelessDinosaur) [inline src/pages/guides/getting-started/installation.mdx:41]:
```suggestion
Alternatively, you can download pre-compiled binaries from the [releases](https://github.com/nitrictech/cli/releases) page and install them manually.
```

File: src/pages/guides/getting-started/installation.mdx (hunk 1 above)
Comment (tjholm) [reply src/pages/guides/getting-started/installation.mdx:33]: This is a good point, worth thinking about as brew can also be used on Linux as well if users choose.

File: src/pages/guides/getting-started/installation.mdx [hunk 3]
```diff
@@ -15,34 +15,58 @@ Please follow these links to the official installation steps for each.
...
-`nitric` is available via [scoop](https://scoop-docs.vercel.app/)
-
-```
+```bash {{ title: 'Windows', label: 'nitric is available via <a href="https://scoop.sh/">scoop</a>' }}
 scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
 scoop install nitric
 ```
 
-### Linux
+```bash {{ title: 'Linux', label: 'nitric can be downloaded as a scripted install via curl.' }}
```
Comment (tjholm) [inline src/pages/guides/getting-started/installation.mdx:29]: Not sure about the labels.
image

The text description is a bit superfluous.

File: src/pages/guides/getting-started/installation.mdx (hunk 1 above)
Comment (davemooreuws) [reply src/pages/guides/getting-started/installation.mdx:33]: ^ yeah I added it for linux users. But then I guess they have the install script too. I can move it as a part of the linux one?

File: src/pages/guides/getting-started/installation.mdx (hunk 3 above)
Comment (davemooreuws) [reply src/pages/guides/getting-started/installation.mdx:29]: Agree, it should be fairly obvious. I added them so it mapped one to one to the old version of doc, but happy to remove.

**Review by HomelessDinosaur — APPROVED** [review]

**Review by tjholm — APPROVED** [review]

**Pull request merged**